# Scene frames
table_reference_frame: "world"
object_reference_frame: "world"

# Collision object for picking
# CYLINDER object specifications
cylinder_name: "cylinder"
cylinder_dimensions: [0.03, 0.03] # [height, radius] threshold is .1 m tall .03 meters radius
cylinder_pose: [0.5, -0.25, 0, 0, 0, 0] # [0.5, -0.25, 0.0, 0, 0, 0]

# Collision object for picking
# CYLINDER object specifications
box_name: "box"
box_dimensions: [0.02, 0.02, 0.25] # [lengthd, width, height] threshold .06 meters wide, .1 m tall and 
box_pose: [0.6, -0.25, 0.0, 0, 0, 0]

# Table model
table_name: "table"
table_dimensions: [0.4, 0.5, 0.1]  # [length, width, height]
table_pose: [0.5, -0.25, 0, 0, 0, 0]